src/marine/viirs_modis_l3_oc2ioda.py

```diff
@@ -67,9 +67,14 @@ def _read(self):
         lats = lats.ravel()[mask]
 
         # get global attributes
-        for v in ('platform', 'instrument', 'processing_version',
-                  'time_coverage_start'):
-            GlobalAttrs[v] = ncd.getncattr(v)
+#        for v in ('platform', 'instrument', 'processing_version',
+#                  'time_coverage_start'):
+#            GlobalAttrs[v] = ncd.getncattr(v)
```

Comment:
@nicholasesposito I recalled during code sprint, platform and sensor use the integer ID rather than string, if you follow that way, please check and see they are integer ID.